hw/top_earlgrey/data/pins_cw310.xdc

```diff
@@ -62,14 +62,14 @@ set_property -dict { PACKAGE_PIN AF10 IOSTANDARD LVCMOS18 PULLTYPE PULLUP } [get
 set_property -dict { PACKAGE_PIN AE11 IOSTANDARD LVCMOS18 } [get_ports { SPI_HOST_CS_L }]; #CSB (USR_SPI0CS)
 
 ## OTHER IO
-set_property -dict { PACKAGE_PIN A8  IOSTANDARD LVCMOS33 } [get_ports { IOC2 }]; #USERIOB-9
```

Comment:
I believe `IOC2` has been removed by accident. instead, `IOC4` should be removed, right? this might be causing the FPGA DRC failure on the CW310.

Reply:
Thanks for noticing!